service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/model/GpcAccessDocumentPayload.java

```diff
@@ -0,0 +1,11 @@
+package uk.nhs.adaptors.gp2gp.gpc.model;
+
+import lombok.Data;
+
+@Data
+public class GpcAccessDocumentPayload {
```

Comment:
This is a FHIR resource. Use HAPI instead of creating your own class?

Reply:
I forgot to remove this class. It was created just for testing purposes. Sorry.